cmd/kubeadm/app/cmd/reset.go

```diff
@@ -215,7 +215,7 @@ func resetWithCrictl(execer utilsexec.Interface, dockerCheck preflight.Checker,
 			resetWithDocker(execer, dockerCheck)
 			return
 		}
-		sandboxes := strings.Split(string(output), " ")
+		sandboxes := strings.Split(string(output), "\n")
```

Comment:
I think it will make more sense to use `strings.Fields()` to split on any whitespace delimiter